Classes/Emogrifier.php

```diff
@@ -1077,6 +1078,14 @@ function ($matches) use (&$media) {
             $cssWithoutComments
         );
 
+	//Keep @imports
```

Comment:
This comment is a smell that the additional code should go into a separate method.